cloud/amazon/ecs_taskdefinition.py

```diff
@@ -142,12 +169,12 @@ def main():
 
     argument_spec = ec2_argument_spec()
     argument_spec.update(dict(
-        state=dict(required=True, choices=['present', 'absent'] ),
-        arn=dict(required=False, type='str' ),
-        family=dict(required=False, type='str' ),
-        revision=dict(required=False, type='int' ),
-        containers=dict(required=False, type='list' ),
-        volumes=dict(required=False, type='list' )
+        state=dict(required=True, choices=['present', 'absent']),
+        arn=dict(required=False, type='str'),
+        family=dict(required=False, type='str'),
+        revision=dict(required=False, type='int'),
+        containers=dict(required=False, type='list'),
+        volumes=dict(required=False, type='list')
```

Comment:
Let's get this merged asap rather than more changes which means more reviews.

I am happy to update https://github.com/ansible/ansible-modules-extras/pull/3016 once merged to add the additonal params